spec/controllers/ems_block_storage_controller_spec.rb

```diff
@@ -30,4 +30,13 @@
       end
     end
   end
+
+  describe '#show_list' do
+    render_views
+
+    it 'renders the Block Storage Managers Toolbar' do
+      expect(ApplicationHelper::Toolbar::GtlView).to receive(:definition)
```

Comment:
;-) you are testing the view toolbar, and that is what you probably wanted here, but you are not testing for the "Block Storage Managers" toolbar (as the message says)

But I'd prefer the message to match the actual test.

You can test for both toolbars, in the same test. Just add another line:

```
expect(ApplicationHelper::Toolbar::Block....something).to receive(:definition)
```

Reply:
Ahh, you are right. I should add this to the test. Thank you very much! ;)